### PR TITLE
Add piped list panel to piped overview dashboard

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

Screen Shot 2021-09-09 at 16 30 15

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: manifests/pipecd/grafana-dashboards/piped/overview.json [hunk 1]
```diff
@@ -153,10 +160,119 @@
...
+                      "result": {
+                        "index": 0,
+                        "text": "-"
+                      }
+                    },
+                    "type": "special"
+                  }
+                ]
+              }
+            ]
```
Comment (nakabonne) [inline manifests/pipecd/grafana-dashboards/piped/overview.json:237]: nit: I would prefer to change the header "piped" to "id" but it's on you

image

Comment (khanhtc1202) [reply manifests/pipecd/grafana-dashboards/piped/overview.json:237]: Sure, lets me adopt it. 👍

Comment (khanhtc1202) [reply manifests/pipecd/grafana-dashboards/piped/overview.json:249]: addressed by https://github.com/pipe-cd/pipe/pull/2449/commits/27c044dc4faadff898a974b3d53424b9eaa62d0a 🙏

**Comment by nakabonne** [comment]
👍 
/lgtm

**Comment by nghialv** [comment]
Nice.
/approve

**Pull request merged**